### DISCUSSION TITLE
Button Group Dropdown

### DISCUSSION THREAD

**Discussion by trippo** [rappasoft/laravel-livewire-tables#1294] [Feature Requests]
Is it possible to have an action buttons dropdown on table rows like Nova or another administration panel?
<img width="126" alt="image" src="https://github.com/rappasoft/laravel-livewire-tables/assets/497169/cdc36793-abae-4fa3-b641-821e6834dca3">

**Comment by lrljoe** [comment]
Yes, please check the docs site for explanation on how this works!

**Reply by trippo** [reply]
I can't find it on Doc... I saw bulk actions dropdown (but I need a dropdown for each row) or Configurable Areas (but it is out of table rows)

**Comment by trippo** [comment]
I tried with [component column](https://rappasoft.com/docs/laravel-livewire-tables/v2/columns/other-column-types#content-component-columns) and i can do what i want 
<img width="224" alt="image" src="https://github.com/rappasoft/laravel-livewire-tables/assets/497169/e8834bb8-186f-4f87-acb2-93904d504dfe">


But I prefer a solution like [button groups](https://rappasoft.com/docs/laravel-livewire-tables/v2/columns/other-column-types#content-button-group-columns) where the buttons in that list are dropdown items

**Comment by lrljoe** [comment]
In this example, you can just use a custom view on a standard column, and you can present it however you like.
